src/sentry/api/utils.py

```diff
@@ -79,9 +76,4 @@ def get_date_range_from_params(params, optional=False, validate_window=True):
     if start > end:
         raise InvalidParams('start must be before end')
 
-    if validate_window:
-        delta = end - start
-        if delta > MAX_STATS_PERIOD:
-            raise InvalidParams(INVALID_PERIOD_ERROR)
```

Comment:
If the requested range is bigger than `MAX_STATS_PERIOD` should we return a tuple of that start,end? Right now it looks like we'd be returning a range with more than 90 days in it.

Reply:
I'm not sure that we actually want to always clamp this to 90 days. This isn't actually the same as many people's retention period anyway.

Reply:
Ok. Its not like a longer window will let them read data that has been deleted as it rolls out of the retention window.